src/Underworld.ts

```diff
@@ -3443,6 +3443,7 @@ ${CardUI.cardListToImages(player.stats.longestSpell)}
     }
   }
   // TODO - This should factor in fortify, debilitate, bloat explosions, etc.
+  // Any reason we dont fully simulate the enemy turn and store relevant results?
```

Comment:
I suspect the enemy turn would take too long to calculate at prediction speeds.  Due to all of them processing collisions and pathing etc